app/models/ahoy_event.rb

```diff
@@ -0,0 +1,3 @@
+class AhoyEvent < ApplicationRecord
+  acts_as_list
```

Comment:
Do we need this?

Reply:
No good catch - I removed the acts as list.